lunar-phase-simulator/index.html

```diff
@@ -4,7 +4,7 @@
         <meta charset="UTF-8" />
         <title>Lunar Phase Simulator</title>
  
-        <link rel="stylesheet" href="https://stackpath.bootstrapcdn.com/bootstrap/4.1.2/css/bootstrap.min.css" integrity="sha384-Smlep5jCw/wG7hdkwQ/Z5nLIefveQRIY9nfy6xoR1uRYBtpZgI6339F5dgvm/e9B" crossorigin="anonymous">
+        <link rel="stylesheet" href="../css/bootstrap.min.css">
```

Comment:
The relative bootstrap include does not work in my local dev env with `npm run serve`

Reply:
currently `npm run serve` is broken.. there's something wrong with webpack-serve. I still develop following the instructions in the readme, with `npm run dev` combined with a local webserver to get around the asset loading issue.

I switched to a local bootstrap because bootstrapcdn was actually down yesterday... or for whatever reason not loading for me, breaking the interactives even in dev mode.